core/src/main/java/org/apache/accumulo/core/client/impl/ConnectionInfoImpl.java

```diff
@@ -38,7 +39,7 @@ public String getInstanceName() {
   }
 
   @Override
-  public String getZookeepers() {
+  public String getZooKeepers() {
```

Comment:
Do we have a consistent way to refer to "zookeepers"?  Camel case or one word?

Reply:
While I would prefer `Zookeeper`, `ZooKeeper` is how the project refers to itself.